src/app/globals.css

```
@@ -1,29 +1,35 @@
 }
 
 body {
   background: var(--background);
   color: var(--foreground);
-  font-family: Arial, Helvetica, sans-serif;
+  /* Geist 는 subsets:['latin'] 이라 한글 글리프가 없음. 라틴만 Geist 가 받고
+     한글은 뒤 폴백으로 넘어감. Pretendard 는 설치돼 있을 때만 쓰이고,
+     없으면 OS 기본 고딕(macOS · Windows 순)으로 떨어짐. */
+  font-family:
+    var(--font-geist-sans),
+    'Pretendard',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)